### PR TITLE
Makes the bluespace miner unorderable (and removes it from the raptor qm closet)

### PR DESCRIPTION
**Pull request by Paxilmaniac**
## About The Pull Request

See title

## How This Contributes To The Nova Sector Roleplay Experience

We have arcmining now which literally teleports resources from lavaland to the station, we don't need the BS miner to entirely overstep that whole system anymore.

I'm not deleting the entire thing *yet* because tarkon still uses it.

## Proof of Testing

If all the checkmarks below are green then there's not much else I could prove to you.

## Changelog

:cl:
del: Bluespace miner boards can no longer be bought
/:cl:

### CODE REVIEW

**Comment by Stalkeros2** [comment]
Give Tarkon thumpers and remove BSMs entirely.

**Comment by Zergspower** [comment]
The bsminer was made for mostly lowpop situations wasnt it? When there aren't miners

**Comment by Zenitheevee** [comment]
I will be exploring alternative solutions to fuel tarkon's material gains.

**Comment by Nerev4r** [comment]
> The bsminer was made for mostly lowpop situations wasnt it? When there aren't miners

Ore thumpers can also do this, to be fair.

**Comment by Paxilmaniac** [comment]
> The bsminer was made for mostly lowpop situations wasnt it? When there aren't miners

With arcmining you get an ore well at roundstart that makes iron and glass forever for free that gets teleported to the station. If you have no miners or even a dude who's willing to go down with a gun and find some wells, then you can just order the materials off the materials market.

**Review by Jolly-66 — APPROVED** [review]
I'm fine with the map changes.

**Comment by Namwty** [comment]
I don't exactly see why we actually are removing it, at least by lookin at PRs like Arcmining etc.

Bluespace miners will not overstep new mining system at all, due to factors of it being overpriced and high effort/space to set up and actually worthless apart from it being "fun" to set up and simply... other methods being better.
Arcminin was balanced by reducing ore spawnrate to account for ore vents, so frankly after manuel test merging in 3 hour rounds... it's safe to say that Arcmining just makes miners work harder for same reward and is generally a timewaste trap... for those who actually don't care for grinding. Not saying that Arcmining is bad, it's fun... but still questionable in many ways.


Reality is already simple
Miners who choose to use thumpers, bluespace miner, Anthills and such... already choose more inefficient path that requires more effort and less reward. So.... why make those options less worthy to do?

Arguably, anthills are better than bluespace miners and Thumpers, apart from fact that they don't make diamonds/bluespace ( which by the way, BS also doesn't make bluespace either )

And finally, are we forgetting that bluespace miners were 30k ( now 10k, btw still worthless )? You can buy 1000 iron for 1k credits using GMM, how in the world would you justify bluespace miners worth or even thumpers worth ( when both of them are just RNG ore farm ( of ores that you might not even need )). We aren't even bringing the cost of generators or other stuff like radiation form some of those generators.



> just order the materials off the materials market.

Just gonna be real
Cargo could have 500k credits, but we still will have:
0 meteor sat crates ordered ( and the be in UTTER DESPAIR about meteor storms, when we could have 2000%+ coverage )
No food when chef is nonexistent
No iron/glass/etc ( even if it costs 1 credit per sheet ) from GMM ordered despite no mats and having credits.
No first aid kits during really bad molds of blobs ( good old days ) and medbay is understaffed
**At best they will order 50 guns and say "yeah, guns will fix everything"**
And frankly, can be said same about Thumpers, why use them when ya could just "Buy stuff from GMM"?


Imo only reason to use thumpers and stuff, is cause they are fun to use.... same can be said for bluespace miner.

But then again, they are useless, so no one ( majority of players ) will miss them, if bluespace miners get removed, so who cares, and tbf I do enjoy your colonial PRs so will trust your decision on removal.



TL;DR

I disagree with removing it for being risk of "overstepping" the new system, cause frankly it's laughable to even consider bluespace miners worth credits/effort/space/time to set up.

But I do agree with removing bluespace miners cause we do have stuff like Thumpers and colonial stuff... that arguably replace them ( even tho they do require land... but that's whatever ).

The only arguable thing that bluespace miners have... is that you don't need lavaland to have em set up but frankly who the hell cares. We don't have people bringing bubblegums to both public mining and normal mining, to justify that trait.

**Comment by drizzlemutt** [comment]
Just gonna throw in my two bits as a prolific user of the BSM as a scientist. 

It'd be a shame if it went away as its currently the only way for researchers to reasonably obtain additional materials and strange rocks without having to leave the station all together, missing out on chances to roleplay with other individuals. The job already has lots of issues with being geared around individual pursuit and lack of interaction, and I think this would just make it overall worse (at least with the things that I personally pursue in my rounds).

If the BSM goes I'd probably never touch xenoarchaeology as much, as the time investment cost just wouldn't be worth it.

Also, for the people saying that the investment isn't worth it for materials, they're mostly correct as the cost is super prohibitive to 99% of players. The cost of one miner at full efficiency really doesn't pay for itself over the course of a round, even with doing xenoarchaeology, but it does provide utility to those that are willing to upfront the cost in terms of a flexible, and handsfree, supply of materials and strange rocks.

As it stands, its just a luxury convenience item and I think that it should stay this way, or be at least reworked into something more in line with arc mining, instead of just flat-out removing it. As removing it would leave this luxury spot vacant that will be most likely be filled with the same thing in the future.

**Comment by Paxilmaniac** [comment]
I'll just redo this then

**Comment by Drayco84** [comment]
> Just gonna throw in my two bits as a prolific user of the BSM as a scientist.
> 
> It'd be a shame if it went away as its currently the only way for researchers to reasonably obtain additional materials and strange rocks without having to leave the station all together, missing out on chances to roleplay with other individuals. The job already has lots of issues with being geared around individual pursuit and lack of interaction, and I think this would just make it overall worse (at least with the things that I personally pursue in my rounds).
> 
> If the BSM goes I'd probably never touch xenoarchaeology as much, as the time investment cost just wouldn't be worth it.
> 
> Also, for the people saying that the investment isn't worth it for materials, they're mostly correct as the cost is super prohibitive to 99% of players. The cost of one miner at full efficiency really doesn't pay for itself over the course of a round, even with doing xenoarchaeology, but it does provide utility to those that are willing to upfront the cost in terms of a flexible, and handsfree, supply of materials and strange rocks.
> 
> As it stands, its just a luxury convenience item and I think that it should stay this way, or be at least reworked into something more in line with arc mining, instead of just flat-out removing it. Removing it would leave this luxury spot vacant that will be most likely be filled with the same thing in the future.

As a miner, I want to say... Xenoarch bags that store lots of strange rocks really should be added to the Miner lockers or better yet, add something that teleports strange rocks right to xenoarch.
Actually, part of that system needs to be overhauled. Add a machine somewhere that teleports strange rocks to xenoarch and gives the miner some mining points per rock. Miner gets paid, xenoarch gets work to do, everybody wins.
